src/Avalonia.Visuals/Rendering/DeferredRenderer.cs

```diff
@@ -243,6 +243,15 @@ private void Render(IDrawingContextImpl context, VisualNode node, IVisual layer,
             {
                 clipBounds = node.ClipBounds.Intersect(clipBounds);
 
+                var np = node.Visual.GetType().GetProperty("Name");
+                var name = (string)np.GetValue(node.Visual);
+
+                if (name == "abcd")
```

Comment:
Can we pick a more descriptive name (and maybe save it in a constant so it isn't a magic string)?
Also, is this used for anything other than debugging?

Reply:
Oh good catch, this is debug code.